infoblox-ipam.go

```diff
@@ -214,22 +215,27 @@ func (ibDrv *InfobloxDriver) requestSpecificNetwork(netview string, subnet strin
 	return network, err
 }
 
-func (ibDrv *InfobloxDriver) RequestNetwork(netconf NetConfig, netviewName string) (network string, err error) {
+func (ibDrv *InfobloxDriver) RequestNetwork(netconf NetConfig, netviewName string) (network string, networks []string, err error) {
```

Comment:
I suggest naming the returned parameters appropriately, to reflect the fact that `network` is a newly allocated network, whereas `networks` is a list of pre-existing network.

Alternatively, since it looks like it return one of `network` or `networks` ONLY, it can just return networks[] plus a flag to indicate wether the result is newly allocated or not.